### DISCUSSION TITLE
Latest update ChromeCast Desktop Audio Streamer 5.2 (12.08.2023) - cannot find .exe file to run this excellent app.

### DISCUSSION THREAD

**Discussion by coppinsuk** [SamDel/ChromeCast-Desktop-Audio-Streamer#148] [Q&A]
Latest update 5.2 - cannot find .exe file to run this excellent app.

Had no problem with previous iterations.  Been using your app. for a number of years successfully, after finding .exe file. Not the easiest for me as I do not have a PhD in IT skills. But this time it is beyond me. What do I do?

Please direct me to the easiest way to "unpack" the update so that I can find the .exe file. An idiots guide please.

Thanking you in anticipation.

Cheers.

**Comment by SamDel** [comment]
Somehow the shortcut in the start menu was removed, in version 5.0 (and later versions).

Sorry about that, and thanks for letting me know!
In [Setup 5.2.1.zip](https://github.com/SamDel/ChromeCast-Desktop-Audio-Streamer/files/12327451/Setup.5.2.1.zip) a shortcut to the application is added in the start menu. Can you try?

**Reply by coppinsuk** [reply]
Thank you for all your work on this excellent app. which I have been enjoying for a number of years. But still cannot find an exe. file. Is that what I should be looking for? What am I doing wrong?

Now deleted all downloaded ChromeCast Desktop Audio Streamer 5.2 files in order to start again.

Will keep trying to find an exe. file - or should I be doing something else? 

Not seen a "start menu" in anything I have downloaded or unpacked.

I do not want to give up.

Cheers.

**Reply by SamDel** [reply]
The shortcut is in the Windows start menu (after installing version 5.2.1), when you press the Windows icon in the lower left corner!?

The .exe file is here:

```
C:\Program Files (x86)\Desktop Audio Streamer\Desktop Audio Streamer
```

**Reply by coppinsuk** [reply]
Been looking for it.

I've changed my Windows 11 start menu back to the "Classic version" which doesn't help. So I am looking for an exe . file - thank you.

I can see a new shortcut in the start menu which takes me to the old? chromecast audio programme - but doesn't go anywhere. Will now delete this and see what happens.

Apparently I needed to D/L some Microsoft software what was missing - don't ask me - didn't keep a record.

This time I was able to d/l the [Setup 5.2.1.zip] file and found two exe. files - one which was 'msi' exe - which confused me - as my laptop is an msi brand - so thought it was not relevant.

I'm getting somewhere, but not sure where - but will keep trying.

Thanks for your forbearance.

No doubt will be in touch, hopefully to let you know that I have been successful.

Cheers.

**Reply by SamDel** [reply]
In the Windows 11 start menu you have to select "All apps" first I think (I tried the 'default' start menu). Or search for "Desktop Audio Streamer" in the start menu.

**Reply by coppinsuk** [reply]
Hooray got there - but not sure how, and whether I will be able to do it again.

Many thanks for you patient guidance.

So pleased that it is now working.

You're a Saint.

Cheers.

**Reply by coppinsuk** [reply]
Hope that I have learnt something.

Cheers.

**Comment by coppinsuk** [comment]
Many thanks for your patience and help in getting the Audio Streamer
upgrade to work.

There was no need to acknowledge my help(?) in your Hit Hub site.

I have now successfully updated it to 5.3.

Take care.

Cheers.

**Comment by coppinsuk** [comment]
By the way it is now successfully playing on my 5 chromecasts (not quite in
sync. alas - another project).

**Comment by FA-Bubba** [comment]
If you place all of your ChromeCasts into a single group, they will be in sync.

**Comment by coppinsuk** [comment]
Thanks for the tip!

I see that there are some guidelines about the procedure of doing so online.

Will try tomorrow.

Thank you.

**Reply by coppinsuk** [reply]
1. Firstly to FA-Bubba, the suggestion about putting all relevant chromecasts in a "group" - got partially there, but couldn't put all the needed ones into a group, because the 'grouping' option was not available for all. I guess I may just have to clean reinstall each of them again into Google Home - perhaps resulting from a number of updates to Google Home app?
 
2. Just been doing some laptop housekeeping in the course of which I deleted the Desktop Audio Streamer 5.3 downloaded file. Now have re-loaded it BUT now it cannot be installed - I keep getting a Windows installer message "The feature you are trying to use is on a CD-rom or other removable disk that is not available"  - with options "OK/Cancel"  then "insert disk and click" and options "use source"  and "browse".  What the hell is going on?  I really thought that I had cracked it! I have about 100TB of music waiting to be thrown to 'AudioStreamer' or are all these files now redundant? Perhaps I should delete all mentions of "ChromeCast Desktop Audio Streamer" and start all over again???

Not sure if all this is intelligible?

If you can muster up some more patience, please would you be so kind as to give me some more guidance.

Thanking you in anticipation.

Cheers.

**Reply by SamDel** [reply]
I don't know why you get the Windows installer message. 
Download Setup 5.3.zip, unzip the two files in it (is not always necessary) and run setup.exe. Is that what you tried?

**Comment by coppinsuk** [comment]
Yes, it does.

Many thanks for getting back to me.

I'll correspond tomorrow - I'm currently doing all my security/health
checks in an attempt to eliminate any exogenous factors that might have
some impact on the issues that I am having.

Be in touch tomorrow.

Thanking you.

**Comment by coppinsuk** [comment]
Still no further on.

Did exhaustive checks of my laptop - disabled firewall and protection
programmes (Windows Defender, Norton, Malwarebytes) and attempted putting
exceptions into them also.

"Windows" is still not liking "*set.exe*".

It used to work OK on my computers for a number of years - but not now!
What's changed?

Is there an alternative?  I desperately need to use "Chromecast Audio" as
it preserves the quality of my recorded music at a high level. As I have
said before, I have about 100TB of mainly flac files that I would like to
access. vlc (VideoLan) is not a contender to Chromecast Audio. PLEX a non
starter - too complicated, too much effort - a tech savvy toy. My main
media player is MusicBee which I used to stream through my Chromecasts -
but not now, alas.

Any suggestions as to what I can do now would be most welcomed.

Please do not give up on me just yet.

**Comment by SamDel** [comment]
I zipped the install folder (version 5.3): [Desktop Audio Streamer.zip](https://github.com/SamDel/ChromeCast-Desktop-Audio-Streamer/files/12346862/Desktop.Audio.Streamer.zip), maybe that works for you?

**Comment by coppinsuk** [comment]
Awesome, thanks!

That works for me - I was really on the point of giving up. This has all
depressed me - made me feel helpless.

Why does this iteration work, whereas the others did not?

During installation I was even given the chance to "offer exceptions" in
Windows Defender - now to reinstall Norton. Hope this all goes OK as well,
or perhaps I shouldn't chance it just yet?

Many thanks for your persistence.

Will let you know how I get on.

**Reply by SamDel** [reply]
Nice! For the other issues you can maybe contact a Windows expert.
